sorting_algos/radix_sort/radix_sort.py

```diff
@@ -0,0 +1,18 @@
+def radix_sort(lst):
+    """Take list of non-negative ints and return sorted list."""
+    if type(lst) is not list:
+        raise TypeError('argument must be type <list>')
+
+    buckets = [[i for i in range(10)] for _ in range(10)]
```

Comment:
This line is unnecessary. You immediately reassign `buckets` to be a list of empty lists in your loop. Also, if your list is precisely ten items long, it will not get sorted due to the condition of your `while` loop.